strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/pypi/PypiBrowsePackageHtmlResponseBuilderTest.java

```diff
@@ -0,0 +1,143 @@
+package org.carlspring.strongbox.controllers.layout.pypi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.carlspring.strongbox.artifact.coordinates.PypiArtifactCoordinates;
+import org.carlspring.strongbox.config.IntegrationTest;
+import org.carlspring.strongbox.providers.io.RepositoryFiles;
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+import org.carlspring.strongbox.storage.repository.Repository;
+import org.carlspring.strongbox.testing.artifact.ArtifactManagementTestExecutionListener;
+import org.carlspring.strongbox.testing.artifact.PypiTestArtifact;
+import org.carlspring.strongbox.testing.repository.PypiTestRepository;
+import org.carlspring.strongbox.testing.storage.repository.RepositoryManagementTestExecutionListener;
+
+import javax.inject.Inject;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * @author ankit.tomar
+ */
+@IntegrationTest
+public class PypiBrowsePackageHtmlResponseBuilderTest
+{
+
+    private static final String REPOSITORY_RELEASES = "pypi-releases-test";
+
+    private static final String STORAGE_PYPI = "storage-pypi-test";
+
+    @Inject
+    private PypiBrowsePackageHtmlResponseBuilder htmlResponseBuilder;
+
+    @Test
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class })
+    public void testNoPackageFoundFlow(@PypiTestRepository(repositoryId = REPOSITORY_RELEASES,
```

Comment:
@anki2189 is it valid case where there is no packages found? I mean that maybe it should be `404` error in this case?

Reply:
@sbespalov Will check this scenario against Pypi server.

Reply:
Will check this scenario against Pypi server

Reply:
testPypi server is also returning `200` in case no package is available.
<img width="1259" alt="Screen Shot 2020-01-07 at 8 08 39 PM" src="https://user-images.githubusercontent.com/7271803/71903328-d6aa7f80-3189-11ea-90e4-9b946aac8501.png">

Reply:
thanks for checking it. What means "Simple Index" here?

Reply:
Request Package name was `hello` . "Simple Index" is default text returned by server which in our case is `Links for packageName`

Reply:
Can you please provide an example of `packageName` that exists?

Reply:
<img width="1280" alt="Screen Shot 2020-01-07 at 8 55 49 PM" src="https://user-images.githubusercontent.com/7271803/71906543-44f24080-3190-11ea-972c-8fdbf94992fa.png">

Reply:
so it's differs for not existing package and existiong one, right?

Reply:
Yes.

Reply:
Thanks for confirming. Can you please make it same way for us? In this case we won't need the `packageName` parameter and can take it from `ArtifactCoordinates` as suggested above.

Reply:
@sbespalov : What should be `<title>` value in case package is not found. Shared screenshots are from test server , I am not sure if it is good idea to do the same.?

Reply:
`Nof found` probably would be fine.

Reply:
Fixed.

Reply:
thanks!